README.md

```diff
@@ -35,7 +35,7 @@ A default config file will be created for you, without extra charges, into the p
 ## Hooks with other plugins
 
 * [GriefPrevention](https://www.spigotmc.org/resources/griefprevention.1884/) RTPs can take claims in consideration.
-  * Works with any other claim plugin too without any hooks!
+* Works with any other claim plugin too without any hooks!
```

Comment:
I purposely indented this but fair enough

Reply:
I though you did a mistake, my bad.